realm/realm-library/src/main/java/io/realm/internal/SharedRealm.java

```diff
@@ -218,6 +245,7 @@ public boolean isEmpty() {
 
     public void refresh() {
         nativeRefresh(nativePtr);
+        invokeListenerIfSchemaChanged();
```

Comment:
Can Realm Object Store's `Realm::add_schema_change_handler()` be used for anything useful here?